### DISCUSSION TITLE
Lots of parallel inserts with SQL: commands out of sync. Did you run multiple statements at once?

### DISCUSSION THREAD

**Discussion by mehdiym** [aarondl/sqlboiler#923] [Q&A] [answered]
Hello,

I'm trying to do lots of inserts in goroutines but if I have too many routines (~1000) I get that error.
I tried to play with SQL client functions SetMaxOpenConns and SetMaxIdleConns but without success.
The SQL driver and sqlboiler aren't supposed to handle lots of parallel requests and to be thread safe?

Generated Insert code returning the error:
```
	result, err := exec.ExecContext(ctx, cache.query, vals...)
```

**Comment by aarondl** [comment, marked as answer]
This error message doesn't appear in sqlboiler nor in the Go standard library that I could see with a quick grep, it's probably coming from whatever sql driver you're using. Keep in mind that the driver you use is completely up to you and has nothing to do with sqlboiler and so it is not something we'll help to resolve.

**Reply by mehdiym** [reply]
Thank you @aarondl for your answer. I'm using the standard sql go driver. I'll go search for driver solutions then.

**Reply by mehdiym** [reply]
I solved my issue by upgrading my MariaDB.
I had v10.3 (debian buster) and I upgraded to v10.5 (last stable).
